src/vzlogger.cpp

```diff
@@ -105,6 +105,45 @@ const char *long_options_descs[] = {
 	NULL /* stop condition for iterator */
 };
 
+void openLogfile(){
+	m_log.lock();
+	FILE *logfd = fopen(options.log().c_str(), "a");
+
+	if (logfd == NULL) {
+		print(log_alert, "Cannot open logfile %s: %s", (char *)0, options.log().c_str(),
+			  strerror(errno));
+		return EXIT_FAILURE;
+	}
+
+	if (gStartLogBuf) {
+		// log current console output to logfile as we missed the start
+		fprintf(logfd, "%s", gStartLogBuf->str().c_str());
+		auto temp = gStartLogBuf;
+		gStartLogBuf = 0;
+		delete temp;
+	}
+
+	options.logfd(logfd);
+	m_log.unlock();
+	print(log_debug, "Opened logfile %s", (char *)0, options.log().c_str());
+}
+
+void closeLogFile(){
+	m_log.lock();
+	if (options.logfd()) {
+		fclose(options.logfd());
+		options.logfd(NULL);
+	}
+	m_log.unlock();
+}
+
+void signalHandlerReOpenLog(int){
+	print(log_info, "received SIGUSR1, closing logfile for re-opening", "");
```

Comment:
is `log_info` ok, or should this be `log_debug`?

Reply:
I think log_info is fine.